packages/ember/lib/index.js

```diff
@@ -439,6 +440,11 @@ Ember.TextField = TextField;
 Ember.TextArea = TextArea;
 Ember.LinkComponent = LinkComponent;
 
+Object.defineProperty(Ember, '_componentManager', {
```

Comment:
Let’s call it `setComponentManager` here?

Reply:
sans underscore?